spec/ics-002-client-semantics/README.md

```diff
@@ -318,6 +318,8 @@ at a particular finalised height (necessarily associated with a particular commi
 Client types must define functions to authenticate internal state of the state machine which the client tracks.
 Internal implementation details may differ (for example, a loopback client could simply read directly from the state and require no proofs).
 
+- The `delayPeriod` is passed to packet-related verification functions in order to allow packets to specify a period which must pass after a header is verified before the packet is allowed to be processed.
```

Comment:
Is packet defining a `delayPeriod`?

Reply:
`delayPeriod` is being defined upon creation of a connection (In conn open init). Each connection end must have the same `delayPeriod` for a connection to be established. The connection then passes this delayPeriod to the packet verification functions implemented by the on chain light clients

Reply:
I understand now what you mean with packet-related verification functions.

Reply:
Aye. I didn't include that definition in this particular spec document because a client shouldn't care "where" the `delayPeriod` is defined. Maybe that's not so important as to avoid mentioning this though.